src/ui/layout/qgslayoutdesignerbase.ui

```diff
@@ -1085,42 +1086,50 @@
     <string>Normal</string>
    </property>
   </action>
-  <action name="mActionPreviewModeGrayscale">
+  <action name="mActionPreviewModeMono">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>Simulate Photocopy (&amp;Grayscale)</string>
+    <string>Simulate Fax (&amp;Mono)</string>
    </property>
   </action>
-  <action name="mActionPreviewModeMono">
+  <action name="mActionPreviewModeGrayscale">
    <property name="checkable">
     <bool>true</bool>
    </property>
    <property name="text">
-    <string>Simulate Fax (&amp;Mono)</string>
+    <string>Simulate Achromatopsia Color Blindness and Photocopy (&amp;Grayscale)</string>
```

Comment:
```suggestion
    <string>Simulate Achromatopsia Color Blindness/&amp;Grayscale</string>
```

I'd drop the "photocopy" note here -- most of the time when someone is previewing grayscale it's for a grayscale printer, not for photocopy purposes

Reply:
Agreed, modified by 989cd04.